scripts/gendocs/generate

```diff
@@ -76,8 +76,11 @@ genversion() {
 }
 
 # The branches named here exist in the `jetstack/cert-manager` repo.
-genversion "release-1.2" "next-docs"
-genversion "release-1.1" "docs"
+
+#genversion "release-1.4" "next-docs"
+genversion "release-1.3" "docs"
+genversion "release-1.2" "v1.2-docs"
```

Comment:
Does this mean that the 1.2 time frame release process missed a step? (And are the instructions fixed so that 1.4 won't?)

Reply:
This shouldn't be in this PR!